tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -219,6 +219,7 @@ def wait_for_done(self, return_contacts=True):
             from gaiatest.apps.contacts.app import Contacts
             return Contacts(self.marionette)
         else:
-            Wait(self.marionette).until(lambda m: self.apps.displayed_app.name != 'Communications')
+            from gaiatest.apps.contacts.app import Contacts
+            Contacts(self.marionette).wait_to_not_be_displayed()
```

Comment:
No, I just tested myself, this doesn't work with that test.
You should undo the changes in here, I'll be fixing this in https://bugzilla.mozilla.org/show_bug.cgi?id=1207098 .